apps/typegpu-docs/src/content/examples/simple/liquid-glass/index.ts

```diff
@@ -0,0 +1,272 @@
+import tgpu from 'typegpu';
+import * as d from 'typegpu/data';
+import * as std from 'typegpu/std';
+import { sdRoundedBox2d } from '@typegpu/sdf';
+import { loadExternalImageWithMipmaps } from './mipmaps.ts';
+import { fullScreenTriangle } from './common.ts';
+
+const root = await tgpu.init();
+const device = root.device;
+const canvas = document.querySelector('canvas') as HTMLCanvasElement;
+const context = canvas.getContext('webgpu') as GPUCanvasContext;
+const presentationFormat = navigator.gpu.getPreferredCanvasFormat();
+
+const mousePosUniform = root.createUniform(d.vec2f);
+
+const { sampledView, sampler } = await loadExternalImageWithMipmaps(
+  root,
+  '/TypeGPU/plums.jpg',
+);
+
+const Params = d.struct({
+  rectDims: d.vec2f,
+  radius: d.f32,
+  start: d.f32,
+  end: d.f32,
+  chromaticStrength: d.f32,
+  refractionStrength: d.f32,
+  blur: d.f32,
+  edgeFeather: d.f32,
+  edgeBlurMultiplier: d.f32,
+});
+const defaultParams: d.Infer<typeof Params> = {
+  rectDims: d.vec2f(0.13, 0.01),
+  radius: 0.003,
+  start: 0.05,
+  end: 0.1,
+  chromaticStrength: 0.02,
+  refractionStrength: 0.1,
+  blur: 1.2,
+  edgeFeather: 2.0,
+  edgeBlurMultiplier: 0.7,
+};
+
+const paramsUniform = root.createUniform(Params, defaultParams);
+
+context.configure({
+  device,
+  format: presentationFormat,
+  alphaMode: 'premultiplied',
+});
+
+const sampleWithChromaticAberration = tgpu.fn(
+  [d.vec2f, d.f32, d.vec2f, d.f32],
+  d.vec3f,
+)(
+  (uv, offset, dir, blur) => {
+    const red = std.textureSampleBias(
+      sampledView,
+      sampler,
+      uv.add(dir.mul(offset)),
+      blur,
+    );
+    const green = std.textureSampleBias(
+      sampledView,
+      sampler,
+      uv,
+      blur,
+    );
+    const blue = std.textureSampleBias(
+      sampledView,
+      sampler,
+      uv.sub(dir.mul(offset)),
+      blur,
+    );
+    return d.vec3f(red.x, green.y, blue.z);
+  },
+);
```

Comment:
Does this work with Deno?
```suggestion
)((uv, offset, dir, blur) => {
  const red = std.textureSampleBias(
    sampledView,
    sampler,
    uv.add(dir.mul(offset)),
    blur,
  );
  const green = std.textureSampleBias(
    sampledView,
    sampler,
    uv,
    blur,
  );
  const blue = std.textureSampleBias(
    sampledView,
    sampler,
    uv.sub(dir.mul(offset)),
    blur,
  );
  return d.vec3f(red.x, green.y, blue.z);
});
```